charts/k8s-monitoring/destinations/otlp-values.yaml

```diff
@@ -229,6 +229,24 @@ processors:
       resource: []
       span: []
       spanevent: []
+
+  filter:
```

Comment:
```suggestion
  filters:
```
we should match what we have in feature-application-observability/values.yaml

Reply:
This one I agree with.